packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -18,13 +18,13 @@ class Plugin {
     required this.dependencies,
     required this.isDirectDependency,
     this.implementsPackage,
-  }) : assert(name != null),
-       assert(path != null),
-       assert(platforms != null),
-       assert(defaultPackagePlatforms != null),
-       assert(pluginDartClassPlatforms != null),
-       assert(dependencies != null),
-       assert(isDirectDependency != null);
+  })  : assert(name != null),
```

Comment:
can you revert the formatting only diffs to this file? They are not all in line with our [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting) and make this harder to review

Reply:
reverted them